src/services/notification/discord.rs

```diff
@@ -12,12 +12,12 @@ use crate::{
 	services::notification::{NotificationError, Notifier},
 };
 
-use super::BaseWebhookNotifier;
+use super::WebhookNotifier;
```

Comment:
Can we import this from the import above? I guess under `services:notification::{WebhookNotifier}`